config/schedule.rb

```diff
@@ -131,6 +133,11 @@
   ruby_script_blockchain 'get_leaders_schedule.rb'
 end
 
+every 1.day, at: '5:10am', roles: [:background] do
+  ruby_script_archives 'archive_ping_thing_stat.rb'
+  ruby_script_archives 'archive_ping_thing.rb'
```

Comment:
There's missing argument - see archive script
without argument it only creates archives, but doesn't remove pings from original table